src/bracketology_utilities.py

```diff
@@ -0,0 +1,145 @@
+"""Bracketology Utilities."""
+
+import random
+
+
+class Team:
+    """Class representing a team in a tournament."""
+
+    def __init__(
+        self, name: str, seed: str, region: str, win_probs: dict[str, float]
```

Comment:
If what you want here is a mapping from a fixed set of categories to floats, look into using an enum: https://docs.python.org/3/library/enum.html